test/apps/CMakeLists.txt

```diff
@@ -285,48 +304,48 @@ teca_add_app_test(test_temporal_reduction_app_monthly_maximum
     COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/test_temporal_reduction_app.sh
     ${CMAKE_BINARY_DIR}/${BIN_PREFIX} ${TECA_DATA_ROOT}
     "prw_hus_day_MRI-CGCM3_historical_r1i1p1_19500101-19501231\\.nc" prw
-    monthly maximum 7 0
+    monthly maximum -1 7 0
```

Comment:
@burlen wouldn't it be `monthly maximum 7 -1 0` what you mean?

Reply:
yes, sorry about that one!

Reply:
No problem, I will fix it.

Reply:
Thank you, sorry for the trouble